app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :increment_post_likes_counter
```

Comment:
- [ ] Good  use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) :+1: